examples/terminal/src/index.ts

```diff
@@ -10,7 +10,7 @@ import { DockPanel, Widget } from '@phosphor/widgets';
 
 import { TerminalSession } from '@jupyterlab/services';
 
-import { Terminal } from '@jupyterlab/terminal';
+import { Terminal } from '@jupyterlab/terminal/lib/widget';
```

Comment:
Adding any sort of import of `Terminal` caused the lazy loading to be counterfeited. Given this is the only change required, I am fine with this.

Reply:
This is back to the way it was.

The difference with the vega invocation is that there we don't use anything from it other than types... webpack doesn't even see it.